rhizome/bin/setup-spdk

```diff
@@ -25,6 +25,7 @@ end
 
 r "sudo --user=#{q_user} mkdir -p #{q_hugepages_dir}"
 r "mount -t hugetlbfs -o uid=#{q_user},size=1G nodev #{q_hugepages_dir}"
+r "echo 'nodev /home/spdk/hugepages hugetlbfs uid=spdk,size=1G 0 0' >> /etc/fstab"
```

Comment:
does a systemd .mount unit work for this? seems like that's preferred these days.